.github/workflows/release-sdk.yml

```diff
@@ -187,22 +253,48 @@ jobs:
         with:
           password: ${{ secrets.PYPI_TOKEN }}
 
+  publish-release-notes:
+    name: Publish Release Notes
+    needs: pypi-publish
+    runs-on: ubuntu-latest
+    if: ${{ inputs.update-changelog }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: release-${{ github.event.inputs.version }}
+      - name: Create Source Archives
+        run: |
+          git archive --format=zip --output=wandb-${{ github.event.inputs.version }}.zip HEAD
+          git archive --format=tar.gz --output=wandb-${{ github.event.inputs.version }}.tar.gz HEAD
+      - name: Extract release notes and set output
```

Comment:
Should we also link to the version in pypi?

Reply:
ah that's a nice idea!